doc/tutorials/cloud/01_import_data.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">One of the options of doing this are parquet-tools library wrapped into docker container. To use it, you need to download one of parquet files locally, then run this docker container against this file. Using the same container, you can also peek into parquet files and looks its actual data. 
</pre><p>Side Note: maybe having a function to create the table definition would be interesting as a feature.</p><p> </p><p>Typos:</p><ul><li>One of the options <strong>for</strong> doing this <strong>is</strong> parquet-tools library wrapped into <strong>a</strong> <a href="https://hub.docker.com/r/nathanhowell/parquet-tools" target="_blank">docker container</a>.</li><li>Using the same container, you can also peek into parquet files and <strong>look at</strong> its actual data.</li><li>From this schema<strong>,</strong> we see that all the columns in parquet file <strong>have</strong> string type and optional (nullable).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<blockquote>maybe having a function to create the table definition would be interesting as a feature.</blockquote><p>This was my first suggestion to Muhammet when I read cloud-storage. But it is not that easy as udfs cannot executed DDL expressions (as far as I was told).</p><p> </p><p>But of course, it would be much better for the user to be able just import parquet files into the new table without dealing with schema.</p>

Reply:
<p>Typos were fixed</p>

Reply:
<p>ah, in my typo fix is another type</p><p> </p><p>One of the options for doing this is <strong>the</strong> parquet-tools library wrapped into a <a href="https://hub.docker.com/r/nathanhowell/parquet-tools" target="_blank">docker container</a>.</p>

Reply:
<p>Regarding the table creation, I have an idea. I will explain it later.</p>